Source/ThirdParty/libwebrtc/Source/third_party/libyuv/BUILD.gn

```diff
@@ -29,10 +29,18 @@ config("libyuv_config") {
       ldflags = [ "-Wl,--dynamic-linker,/system/bin/linker64" ]
     }
   }
-  defines = []
+
+  # Define CHROMIUM to tell cpu_id to avoid sandbox unsafe system calls.
+  defines = [ "CHROMIUM" ]
```

Comment:
do we need to do something similar here?

Reply:
I don;t think so, I'll compile out cpu_id.cc, this is not useful.